org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/problem/messages.properties

```diff
@@ -287,9 +287,9 @@
 264 = Invalid high surrogate: must be within 0xD800 and 0xDBFF
 265 = Unnecessary $NON-NLS$ tag
 266 = Invalid binary literal number (only '0' and '1' are expected)
-267 = Binary literals can only be used with source level 1.7 or greater
+267 = Binary literals can only be used with source level 1.8 or greater
 268 = Underscores have to be located within digits
-269 = Underscores can only be used with source level 1.7 or greater
+269 = Underscores can only be used with source level 1.8 or greater
```

Comment:
Again, these messages may never surface, but the rewording looks misleading as it appears to make a general statement about the java level at which the feature got introduced.

Reply:
I was thinking about that too, but since we don't accept the targets below `1.8` it would not make sense to give an error saying you should use `1.7`.

Reply:
I will restore 1.7